Access Engineering.js

```diff
@@ -0,0 +1,333 @@
+	"label": "Access Engineering",
+	"creator": "Vinoth K - highwirepress.com",
+	"target": "^https?://www.accessengineeringlibrary.com/content/(book|chapter|case-study|video|calculator|tutorial)",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2022-06-22 08:15:39"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2020-2021 Vinoth K - highwirepress.com
+	
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	let title = attr(doc, 'meta[name="citation_title"]', 'content');
+	if (title) {
+		if (doc.querySelector('meta[name="citation_isbn"]')) {
+			let bookTitle = attr(doc, 'meta[name="citation_book_title"]', 'content');
+			if (!bookTitle || title == bookTitle) {
+				return "book";
+			}
+			else {
+				return "bookSection";
+			}
+		} else if(url.includes('content/video/')) {
+			return 'videoRecording'; 
+		} else {
+			return "journalArticle";
+		}
+	}
+	else if (getSearchResults(doc, true)) {
+		return "multiple";
+	}
+	return false;
+}
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	var rows = doc.querySelectorAll('.results-item a[href]');
+	for (let row of rows) {
+		let href = row.href;
+		let title = ZU.trimInternal(row.textContent);
+		if (!href || !title) continue;
+		if (checkOnly) return true;
+		found = true;
+		items[href] = title;
+	}
+	return found ? items : false;
+}
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+		Zotero.selectItems(getSearchResults(doc, false), function (items) {
+			if (items) ZU.processDocuments(Object.keys(items), scrape);
+		});
+	}
+	else {
+		scrape(doc, url);
+	}
+}
+
+function scrape(doc, url) {
+	// EM is a provides better rule. Most of the empty field 
+	// filled from EM and its missing editions 
+	// for books and books chapter page - so we'll fill
+	// those in manually.
+	var translator = Zotero.loadTranslator('web');
+	// Embedded Metadata
+	translator.setTranslator('951c027d-74ac-47d4-a107-9c3069ab7b48');
+	translator.setDocument(doc);
+
+	translator.setHandler('itemDone', function (obj, item) {
+		// Edition
+		let edition = ZU.xpathText(doc, '//meta[@name="citation_edition"]/@content');
+		if(edition) item.edition = edition;
```

Comment:
```suggestion
		if (edition) item.edition = edition;
```

Run `npm i && npm run lint -- "Access Engineering.js" --fix` to fix all style problems.

Reply:
Formatting style Fixed.